src/components/pages/Home/RenderHomePage.js

```diff
@@ -1,47 +1,22 @@
+import 'antd/lib/menu/style/index.css';
 import React from 'react';
 import { Link } from 'react-router-dom';
 import { Button } from 'antd';
 import AppHeader from '../../common/AppHeader';
+import AppMenu from '../../common/AppMenu';
 
 function RenderHomePage(props) {
   const { userInfo, authService } = props;
   return (
     <div>
       <AppHeader />
-      <div>
+      <div className="content-conatiner">
```

Comment:
this one is also misspelled- they're both spelled the same way but it would be good to have them correct for future use.